README.md

```diff
@@ -32,6 +32,18 @@ slack_webhook_path=""
 # Create your own credentials here https://firebase.google.com/
 author_firebase_project_id=""
 author_firebase_api_key=""
+
+# A service account is required to verify Firebase tokens in Author's API.
+# A key file can be generated for the service account by following the instructions at https://firebase.google.com/docs/admin/setup#initialize_the_sdk
+# The following variable needs to be set and should be a path to the JSON key file.
+author_firebase_service_account_key=""
```

Comment:
its unclear here that the contents of the s3 bucket will be synced to the container under the `/secrets` folder. (https://github.com/ONSdigital/eq-author-app/blob/master/eq-author-api/docker-entrypoint.sh#L5)
So we should make it clear that this path need to be the absolute path of that file inside the container after the sync